src/main/java/Lotto.java

```diff
@@ -0,0 +1,71 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Lotto {
+    private final List<Integer> nums;
```

Comment:
```suggestion
    private final List<LottoNumber> nums;
```

콜렉션의 원시값도 LottoNumber와 같은 객체로 포장해 본다.
이때 얻을 수 있는 이점은 무엇인가?